components/web-search.tsx

```
@@ -0,0 +1,174 @@
+      <div className="text-sm text-muted-foreground">
+        No search results found.
+      </div>
+    );
+  }
+
+  return (
+    <Sheet>
+      <SheetTrigger asChild>
+        <Button size='sm' variant='secondary' className='rounded-full'>Sources
+          <div className='flex items-center -space-x-1.5'>
+            {results.filter(result => !!result.favicon).slice(0, 3).map(result => <img src={result.favicon} className='size-5 rounded-full border border-border bg-secondary' />)}
```

ESLint

[error] Missing "key" prop for element in iterator

(react/jsx-key)